README.md

```diff
@@ -147,6 +147,14 @@ A curated list of awesome Java frameworks, libraries and software.
 * [Cobertura](http://cobertura.github.io/cobertura/) - Relies on offline (or static) bytecode instrumentation and class loading to collect code coverage metrics; GPLv2 licensed.
 * [JaCoCo](http://eclemma.org/jacoco/) - Framework that enables collection of code coverage metrics, using both offline and runtime bytecode instrumentation; prominently used by EclEmma, the Eclipse code-coverage plugin.
 
+## Code Generators
+
+*Code that generates bug-prone boilerplate code.*
+
+* [FreeBuilder](https://github.com/google/FreeBuilder) - Automatic generation of the Builder pattern for Java 1.6+ with Java 8 support.
+* [Auto](https://github.com/google/auto) - Generates factory, service, and value classes and provides other utils.
+* [Lombok](https://github.com/rzwitserloot/lombok) - Generates Java boilerplate.
```

Comment:
We have Lombok already added. Therefore you should move the entry:

`Lombok - Code-generator which aims to reduce the verbosity.` 

out of `Development`.